### ISSUE TITLE
Error: Cannot find module 'numbro'

### ISSUE THREAD

**Issue by vBm** [BenjaminVanRyseghem/numbro#320]
Tried to use latest numbro (2.0.2) by doing as per documentation:
`var numbro = require("numbro");`

This shows the error:
```
Error: Cannot find module 'numbro'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
```

Doing `var numbro = require("numbro/dist/numbro");` makes it work again.

**Comment by BenjaminVanRyseghem** [comment]
my bad, a typo in `package.json` 😕

**Comment by BenjaminVanRyseghem** [comment]
should be fixed in 2.0.3.

Thanks!
